src/gamestate.ts

```diff
@@ -116,7 +120,7 @@ export class GameState {
 		try {
 			renderer.render_jays();
 		} catch (err) {
-			console.error(err);
+			// console.error(err);
```

Comment:
On aura même plus besoin du try catch plus tard t'inquiètes. C'est juste que le warning me gonfle pour l'instant

Reply:
Quel warning ?

Reply:
Un waring qui est dû au fait que je charge une image déjà chargée dans un canvas créé temporairement pour le preloading

Reply:
Uncaught TypeError: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The provided value is not of type '(CSSImageValue or HTMLImageElement or SVGImageElement or HTMLVideoElement or HTMLCanvasElement or ImageBitmap or OffscreenCanvas)'

Reply:
Le type correspond pourtant contrairement à ce que dis l'erreur, j'ai l'impression qu'il a juste pas compris complètement la manip du preload. J'ai jamais réussi à m'en débarrasser.
Mais de toutes façons tu voulais faire le preload à base de promises, ce serait effectivement pas mal de virer ce preloader un jour, donc bon